R/dnn_estimators.R

```diff
@@ -69,6 +69,8 @@ dnn_classifier <- function(hidden_units,
                            input_layer_partitioner = NULL,
                            config = NULL)
 {
+  params <- as.list(environment(), all = TRUE)
```

Comment:
Could you use some variable name other than `params`? Estimator actually has a `params` field in Python API that we haven't expose in R yet but this i likely to introduce issues.